src/components/overview/AddToCart/SizeAndQuantitySelector.jsx

```diff
@@ -48,39 +48,35 @@ const SizeAndQuantitySelector = ({state, updateState}) => {
 
   return (
     <>
-      {state !== undefined && (
-        <>
-          <div id='quantityOption'>
-            {state.currentSize !== '' && state.currentSize !== 'default' && (
-              quantity.length > 9 ?
-                <div id='enoughStock' style={{color: '#3aba2c'}}>
-                In Stock
-                </div> :
-                <div id='lessThanTen' style={{color: '#f75d5d'}}>
-                Only {quantity.length} available!
-                </div>
-            )}
-          </div>
-          <div id='sizeAndQuantitySelector'>
-            <select onChange={updateSize} id='size'>
-              <option id='SizeDefault' value='default' key='size'>Select Size</option>
-              {sizeAndQuantity !== null && (
-                Object.keys(sizeAndQuantity).map((element, index) => {
-                  return <option id={`size${element}`} key={index}>{element}</option>;
-                })
-              )}
-            </select>
-            <select id='quantity'>
-              {quantity[0] !== undefined ?
-                quantity.map((element, index) => {
-                  return <option id={`quantity${element}`} value={`quantity${element}`} key={index}>{element}</option>;
-                }) :
-                <option id='QuantityDefault' value='default' key='default'>-</option>
-              }
-            </select>
-          </div>
-        </>
-      )}
+      <div id='quantityOption'>
+        {state.currentSize !== '' && state.currentSize !== 'default' && (
+          quantity.length > 9 ?
+            <div id='enoughStock' style={{color: '#3aba2c'}}>
+            In Stock
+            </div> :
+            <div id='lessThanTen' style={{color: '#f75d5d'}}>
+            Only {quantity.length} available!
+            </div>
+        )}
+      </div>
+      <div id='sizeAndQuantitySelector'>
+        <select onChange={updateSize} id='size'>
+          <option id='SizeDefault' value='default' key='size'>Select Size</option>
+          {sizeAndQuantity !== null && (
+            Object.keys(sizeAndQuantity).map((element, index) => {
+              return <option id={`size${element}`} key={index}>{element}</option>;
+            })
+          )}
+        </select>
+        <select id='quantity'>
+          {quantity[0] !== undefined ?
+            quantity.map((element, index) => {
+              return <option id={`quantity${element}`} value={`quantity${element}`} key={index}>{element}</option>;
```

Comment:
Possibly an excessively long line. Maybe split it?

Reply:
I agree with you. I will split them for a better readability.